code-challenges.test.js

```diff
@@ -14,46 +14,272 @@
 
-const numbersArray1 = [6, 7, 8, 9, 10]
+const numbersArray1 = [6, 7, 8, 9, 10];
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+const numbersArray2 = [24, 27, 30, 33, 36];
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
+/*
+---- TEST TEMPLATE ----
+describe("", () => {
+  it(``, () => {
+    expect().toEqual()
+  })
+})
+--------
+// I changed the quotation marks inside of the it function to a template literal, just to reduce the headache of having to change quotation marks inside of it. It seems to work just fine, but please let me know if doing so would be a bad convention!
+
+---- PSEUDOCODE TEMPLATE ----
+--------
+// I created a template for my pseudocode because it helps keep me focused, and also ensures I don't miss anything when I'm planning out my code.
+--------
+//
+//
+--------- FUNCTION INFORMATION:
+Describe what the function will do:
+Function Name:
+Function Parameter(s):
+--------- INPUT / OUTPUT:
+input 1:
+output 1:
+
+input 2:
+output 2:
+--------- STEPS/METHODS:
+1.
+2.
+3.
+4.
+*/
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("isObjectEven", () => {
+  it(`Takes a object as an argument and decides if the number inside it is evenly divisible by three or not.`, () => {
+    const object1 = { number: 15 };
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 };
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 };
+    // Expected output: "-7 is not divisible by three"
+    expect(isObjectEven(object1)).toEqual("15 is divisible by three");
+    expect(isObjectEven(object2)).toEqual("0 is divisible by three");
+    expect(isObjectEven(object3)).toEqual("-7 is not divisible by three");
+  });
+});
+
+// RED:
+// Good fail, error message:  ReferenceError: isObjectEven is not defined
 
 // b) Create the function that makes the test pass.
+/*
+// Before I get into coding this, I need to figure out how to make a function that's able to reliably access the key:value pairs inside of an object. After doing some research, I've discovered some methods that work specifically on objects. The one I'm interested in is.values(). Using this, I should be able to convert just the value of the key:value pairs into an array, which I can then use something like .filter() on for my comparison. To confirm this, I'm going to run a few tests of my own before I get into building the function.
```

Comment:
Thank you for adding this here. I am happy to see your process and thorough research.  Keep in mind, that you can also access the value of an object by using dot notation.  Ex: object1.number would give you 15.